src/journal/journalctl.c

```diff
@@ -117,6 +118,7 @@ static bool arg_force = false;
 static usec_t arg_since, arg_until;
 static bool arg_since_set = false, arg_until_set = false;
 static char **arg_syslog_identifier = NULL;
+static char **arg_exclude_identifier = NULL;
```

Comment:
no destructor was added for this

Reply:
Isn't it the destructor:
https://github.com/systemd/systemd/blob/f5c756a254819a94173f04cae0028868fe1339f0/src/journal/journalctl.c#L2863

Reply:
yes, missed that